### PR TITLE
Update office-365-groups-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
We have tested it in a demo tenant and if a group is created via teams then there is no expiration email triggered instead only a notification shows up in the activity section to renew the group.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/create-groups/office-365-groups-expiration-policy.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ If you have setup retention policy in Security and Compliance center for groups,
 
 ## How and when a Group owner learns if their Groups are going to expire
 
-Group owners will only be notified via email, regardless of how the group was created. That means that even if the group was created via Planner, SharePoint, Teams or any other app, the expiration notifications will always come via email. It's not recommended to enable expiration on a group if your group owner doesn't have a valid email address.
+Group owners will only be notified via email. Even if the group was created via Planner or SharePoint, or any other app except for Teams the expiration notifications will always come via email.If the group was created via Teams the group owner will receive a notification to renew through the activity section. It's not recommended to enable expiration on a group if your group owner doesn't have a valid email address.
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/office-365-groups-expiration-policy.md]:
```suggestion
Group owners will only be notified via email. Even if the group was created via Planner or SharePoint, or any other app except via Teams, the expiration notifications will always come via email. If the group was created via Teams, the group owner will receive a notification to renew through the activity section. It's not recommended that you enable expiration on a group if your group owner doesn't have a valid email address.
```

File: Office365-Admin/create-groups/office-365-groups-expiration-policy.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ If you have setup retention policy in Security and Compliance center for groups,
 
 ## How and when a Group owner learns if their Groups are going to expire
 
-Group owners will only be notified via email, regardless of how the group was created. That means that even if the group was created via Planner, SharePoint, Teams or any other app, the expiration notifications will always come via email. It's not recommended to enable expiration on a group if your group owner doesn't have a valid email address.
+Group owners will only be notified via email. Even if the group was created via Planner or SharePoint, or any other app except for Teams, the expiration notifications will always come via email. If the group was created via Teams, the group owner will receive a notification to renew through the activity section. It's not recommended that you enable expiration on a group if your group owner doesn't have a valid email address.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/create-groups/office-365-groups-expiration-policy.md]: Group owners will only be notified via email, except for groups created via Teams. If the group was created via Planner, SharePoint, or any app, the expiration notifications will always come via email. If the group was created via Teams, the group owner will receive a notification to renew through the activity section. It's not recommended that you enable expiration on a group if your group owner doesn't have a valid email address.

Comment (msbemba) [reply Office365-Admin/create-groups/office-365-groups-expiration-policy.md]: Made changes as per your request.

File: Office365-Admin/create-groups/office-365-groups-expiration-policy.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ If you have setup retention policy in Security and Compliance center for groups,
 
 ## How and when a Group owner learns if their Groups are going to expire
 
-Group owners will only be notified via email. Even if the group was created via Planner or SharePoint, or any other app except for Teams, the expiration notifications will always come via email. If the group was created via Teams, the group owner will receive a notification to renew through the activity section. It's not recommended that you enable expiration on a group if your group owner doesn't have a valid email address.
+Group owners will only be notified via email, except for groups created via Teams. If the group was created via Planner, SharePoint, or any app, the expiration notifications will always come via email. If the group was created via Teams, the group owner will receive a notification to renew through the activity section. It's not recommended that you enable expiration on a group if your group owner doesn't have a valid email address.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/create-groups/office-365-groups-expiration-policy.md]: sorry, that should say "any other app". Thanks!

Comment (msbemba) [reply Office365-Admin/create-groups/office-365-groups-expiration-policy.md]: Corrected that.

**Pull request merged**